cwl/src/main/scala/wdl4s/cwl/Cwl.scala

```diff
@@ -13,12 +13,16 @@ sealed trait Cwl {
 }
 
 case class Workflow(
-  cwlVersion: Option[CwlVersion] = None,
-  `class`: Witness.`"Workflow"`.T = "Workflow".narrow,
+  cwlVersion: Option[CwlVersion] = Option(CwlVersion.default),
```

Comment:
Ok, I changed it to CwlVersion.Version1.

(If CommandLineTool and Workflow are the only objects using it, I guess it's no big deal).